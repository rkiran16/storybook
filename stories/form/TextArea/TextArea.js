import { html } from "lit-html";

export const TextArea = () => {
   return html`
   <div class="form-group">
      <textarea class="form-control" id="exampleFormControlTextarea1" placeholder="Text area example" rows="3"></textarea>
      <label for="floatingInput">Label</label>
   </div>
   `;
}