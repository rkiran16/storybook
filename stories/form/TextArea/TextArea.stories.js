import { TextArea } from "./TextArea";

export default {
   title: "Forms/TextArea"
}

const TextAreaTemplate = args => TextArea(args);

export const Default = TextAreaTemplate.bind({})
Default.parameters = {
  layout: 'centered',
};