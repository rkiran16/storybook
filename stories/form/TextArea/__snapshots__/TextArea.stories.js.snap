// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Forms/TextArea Default 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
   <div class=\\"form-group\\">
      <textarea class=\\"form-control\\" id=\\"exampleFormControlTextarea1\\" placeholder=\\"Text area example\\" rows=\\"3\\"></textarea>
      <label for=\\"floatingInput\\">Label</label>
   </div>
   ",
  ],
  "values": Array [],
}
`;
